intermediate_source/tensorboard_profiler_tutorial.py

```diff
@@ -392,6 +393,110 @@ def train(data):
 #
 # The "Communication Operations Stats" summarizes the detailed statistics of all communication ops in each worker.
 
+######################################################################
+# 7. Additional Practices: Profiling PyTorch on AMD GPUs
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#  
+# The AMD ROCm Platform is an open-source software stack designed for GPU computation, consisting of drivers, development tools, and APIs. 
+# We can run above steps on AMD GPUs. In this section of the tutorial, we will use docker to install PyTorch on ROCm.
+
+
+######################################################################
+# Suppose we create a directory called ``profiler_tutorial``, and save the code in Step 1 as ``test_cifar10.py`` in this directory. 
+# 
+# .. code-block::
+#
+#      mkdir ~/profiler_tutorial
+#      cd profiler_tutorial
+#      vi test_cifar10.py
+#
+# Then pull the docker image and start the container:
+#
+# .. code-block::
+#
+#     docker pull rocm/pytorch-nightly:latest
```

Comment:
- Tutorials should use release images to get repeatable results.
- Why use docker container? Are you saying that wheels published on https://pytorch.org/get-started/locally/ are not usable?

Reply:
sure, I will switch to use wheels published on https://pytorch.org/get-started/locally/.

Reply:
Using docker container is a common practice to isolate development or testing environment. Is it a question about docker container approach itself, or you would like to explain this in the tutorial?